js/index.ts

```diff
@@ -0,0 +1 @@
+export { Varint, Uint64} from "./src/varint";
```

Comment:
Even though I made Uint64 public, that was mostly just to write tests for it. I don't actually want it to be part of the public API, and would love to delete it once TypeScript has support for this:

https://github.com/tc39/proposal-integer